daml-lf/archive/src/main/protobuf/com/daml/daml_lf_dev/daml_lf_1.proto

```diff
@@ -1140,6 +1194,17 @@ message Update {
     Expr key = 2;
   }
 
+  // Try-catch block.
+  message TryCatch {
+    Type return_type = 1;
```

Comment:
```suggestion
    Type type = 1;
```

Reply:
I think `return_type` is less ambiguous than `type`. The exception being caught always has `AnyException` type, but if someone sees `type` they might think it refers to the type of the exception being caught. 

Another alternative is `result_type`.